languages/csharp/concepts/enums/links.json

```diff
@@ -1 +1,18 @@
-[]
+[
+  {
+    "url": "https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum",
+    "description": "enumeration types"
+  },
+  {
+    "url": "https://docs.microsoft.com/en-us/dotnet/api/system.enum.parse?view=net-5.0",
+    "description": "Enum.Parse"
+  },
+  {
+    "url": "https://docs.microsoft.com/en-us/dotnet/api/system.enum.tryparse?view=net-5.0",
+    "description": "Enum.TryParse"
+  },
```

Comment:
I think I would remove this link. It is already in the `about.md` and having just the `Enum.Parse` links is enough I think.

```suggestion
```